### ISSUE TITLE
The program make error with GPU

### ISSUE THREAD

**Issue by panda162** [xingyizhou/CenterNet#531]
**OS：**
Windows10
Pytorch 1.2.0
GeForce GTX 1660
CUDA 10.0
CUDNN：7.6.4

**Problem：**
it can run with cpu, but when i use the order below, it will make error:
python main.py ctdet --exp_id coco_dla --batch_size 8 --master_batch 1 --lr 1.25e-4  --gpus 0

**error information：**
Fix size testing.
training chunk_sizes: [1]
The output will be saved to  G:/CenterNet/src\lib\..\..\exp\ctdet\car
==> initializing car val data.
loading annotations into memory...
Done (t=0.00s)
creating index...
index created!
Loaded val 181 samples
<lib.datasets.dataset_factory.get_dataset.<locals>.Dataset object at 0x000001AE92F63D30>
==> initializing car train data.
loading annotations into memory...
Done (t=0.01s)
creating index...
index created!
Loaded train 1452 samples
<lib.datasets.dataset_factory.get_dataset.<locals>.Dataset object at 0x000001AE92F63D30>
heads {'hm': 1, 'wh': 2, 'reg': 2}
Namespace(K=100, aggr_weight=0.0, agnostic_ex=False, arch='res_18', aug_ddd=0.5, aug_rot=0, batch_size=8, cat_spec_wh=False, center_thresh=0.1, chunk_sizes=[1], data_dir='G:/CenterNet/src\\lib\\..\\..\\data', dataset='car', debug=0, debug_dir='G:/CenterNet/src\\lib\\..\\..\\exp\\ctdet\\car\\debug', debugger_theme='white', demo='', dense_hp=False, dense_wh=False, dep_weight=1, dim_weight=1, down_ratio=4, eval_oracle_dep=False, eval_oracle_hm=False, eval_oracle_hmhp=False, eval_oracle_hp_offset=False, eval_oracle_kps=False, eval_oracle_offset=False, eval_oracle_wh=False, exp_dir='G:/CenterNet/src\\lib\\..\\..\\exp\\ctdet', exp_id='car', fix_res=True, flip=0.5, flip_test=False, gpus=[0], gpus_str='0', head_conv=64, heads={'hm': 1, 'wh': 2, 'reg': 2}, hide_data_time=False, hm_hp=True, hm_hp_weight=1, hm_weight=1, hp_weight=1, input_h=512, input_res=512, input_w=512, keep_res=False, kitti_split='3dop', load_model='', lr=0.000125, lr_step=[90, 120], master_batch_size=1, mean=array([[[0.33154365, 0.3768447 , 0.44005337]]], dtype=float32), metric='loss', mse_loss=False, nms=False, no_color_aug=False, norm_wh=False, not_cuda_benchmark=False, not_hm_hp=False, not_prefetch_test=False, not_rand_crop=False, not_reg_bbox=False, not_reg_hp_offset=False, not_reg_offset=False, num_classes=1, num_epochs=140, num_iters=-1, num_stacks=1, num_workers=0, off_weight=1, output_h=128, output_res=128, output_w=128, pad=31, peak_thresh=0.2, print_iter=0, rect_mask=False, reg_bbox=True, reg_hp_offset=True, reg_loss='l1', reg_offset=True, resume=False, root_dir='G:/CenterNet/src\\lib\\..\\..', rot_weight=1, rotate=0, save_all=False, save_dir='G:/CenterNet/src\\lib\\..\\..\\exp\\ctdet\\car', scale=0.4, scores_thresh=0.1, seed=317, shift=0.1, std=array([[[0.15521921, 0.15147787, 0.14349517]]], dtype=float32), task='ctdet', test=False, test_scales=[1.0], trainval=False, val_intervals=5, vis_thresh=0.3, wh_weight=0.1)       
'cp' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
Creating model...
=> loading pretrained model https://download.pytorch.org/models/resnet18-5c106cde.pth
Setting up data...
==> initializing car val data.
loading annotations into memory...
Done (t=0.00s)
creating index...
index created!
Loaded val 181 samples
==> initializing car train data.
loading annotations into memory...
Done (t=0.01s)
creating index...
index created!
Loaded train 1452 samples
Starting training...
+++++++++++1+++++++++++
ctdet/carE:\Python3\lib\site-packages\torch\nn\_reduction.py:43: UserWarning: size_average and reduce args will be deprecated, please use reduction='sum' instead.   
  warnings.warn(warning.format(ret))
PS G:\CenterNet> 

i found the program is end  when it run  at  "losss.backwrad()"

**Comment by xingyizhou** [comment]
I can not see any issues from the log. How do you know the issue is from "loss.backward()"?

**Comment by panda162** [comment]
> I can not see any issues from the log. How do you know the issue is from "loss.backward()"?

我在loss.backward()地方断点调试，程序执行到这句就自动结束了，不过问题已经解决了,我在训练函数前面加了一句“torch.backends.cudnn.enabled = False”就可以正常用GPU进行训练了，但是具体什么原因目前还不是很清楚。

**Comment by xingyizhou** [comment]
Closing this since the issue is resolved.

**Comment by JOSHUAWENJZ** [comment]
@panda162 Hi ! I just saw your implement on GTX 1660. How many seconds did it cost on per image?
